pre_commit/pre_commit_helper.rb

```diff
@@ -9,7 +9,7 @@ def self.project_type(toplevel = `git rev-parse --show-toplevel`.strip)
     return nil
   end
 
-  def self.check_file?(params)
+  def self.check_file_based_on_extension?(params)
```

Comment:
Assignment Branch Condition size for check_file_based_on_extension? is too high. [16.52/15]